homeassistant/helpers/location.py

```diff
@@ -41,3 +48,61 @@ def closest(
             longitude,
         ),
     )
+
+
+def find_coordinates(
+    hass: HomeAssistantType, entity_id: str, recursion_history: Optional[list] = None
+) -> Optional[str]:
+    """Find the gps coordinates of the entity in the form of '90.000,180.000'."""
+    entity = hass.states.get(entity_id)
```

Comment:
This is a state and not an entity. Please name the variable `state` or `entity_state` or similar.

Reply:
Done. Thank you!